src/pseudo/PspUpf.jl

```diff
@@ -181,6 +182,17 @@ function eval_psp_projector_fourier(psp::PspUpf, i, l, q::T)::T where {T <: Real
     4T(π) * s
 end
 
+# For UPFs, the integral is transformed to the following sum:
+# 4π Σ{k} j_l(q r[k]) (r[k]^2 p_{il}(r[k]) dr[k])
+function eval_psp_pswfc_fourier(psp::PspUpf, i, l, q::T)::T where {T <: Real}
```

Comment:
Ok so the pswfc essentially have the same structure as the projectors. Therefore we should not use prefixes but marker structs for the dispatch (e.g. `struct Proj end; struct PsWfc end`). @azadoks might have some plans for this already in PspIO, so ideally we use something here that can be easily integrated with his ideas.